app-emulation/lxd/lxd-2.16.ebuild

```diff
@@ -45,11 +45,9 @@ LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~amd64"
 
-# Portage complains about zh_Hans missing, but repoman doesn't like it when it's there.
-PLOCALES="de el fr it ja nl ru sr sv tr zh"
 IUSE="+daemon nls test"
 
-# IUSE and PLOCALES must be defined before l10n inherited
+# IUSE must be defined before l10n inherited
 inherit bash-completion-r1 golang-build l10n linux-info systemd user golang-vcs-snapshot
```

Comment:
I thought that `domo` was provided by `l10n`, but now that I look, I see that it isn't.